cla_backend/libs/eligibility_calculator/cfe_civil/savings.py

```diff
@@ -0,0 +1,30 @@
+
+def translate_savings(savings_data):
+    liquid_capital = []
+    non_liquid_capital = []
+    if savings_data.bank_balance > 0:
+        liquid_capital.append(
+            {
+                "value": savings_data.bank_balance / 100,
```

Comment:
I think /100 will result in an int, so missing the pence. How about a function to convert like:
```from decimal import Decimal
(Decimal(bank_balance) / 100).quantize(Decimal(".01"))```

Reply:
cool, I see it is done in db728696